tests/helpers/Participant.ts

```diff
@@ -476,6 +541,21 @@ export class Participant {
      * Hangups the participant by leaving the page. base.html is an empty page on all deployments.
      */
     async hangup() {
+        // do a hangup, to make sure unavailable presence is sent
+        await this.driver.execute(() => APP.conference?.hangup());
+
+        const current = await this.driver.getUrl();
+
+        // let's give it some time to leave the muc, we redirect after hangup so we should wait for the
+        // change of url
+        await this.driver.waitUntil(
+            async () => current !== await this.driver.getUrl(),
```

Comment:
ditto

Reply:
Well this needs the await so it can do the comparison, otherwise will compare string to promise.